src/main/java/codechicken/nei/api/API.java

```diff
@@ -300,7 +300,7 @@ public static void registerStackStringifyHandler(IStackStringifyHandler handler)
      * @param stack the ingredient that can craft recipes (like a furnace or crafting table)
      * @param handler the recipe category handled by the ingredient
      */
-    public static void addRecipeCatalyst(ItemStack stack, Class<? extends IRecipeHandler> handler) {
+    public static void addRecipeCatalyst(ItemStack stack, IRecipeHandler handler) {
```

Comment:
is this one of those "removing explicit implication" things IDEA always wants to do or is it actually changing a public interface?